posthog/hogql_queries/insights/funnels/test/__snapshots__/test_funnel_persons.ambr

```diff
@@ -1,158 +1,44 @@
 # serializer version: 1
-# name: TestFunnelPersons.test_funnel_person_recordings
```

Comment:
Looks like all the `TestFunnelPersons` turned into `TestFunnelPersonsUDF` – is this intended here?

Reply:
Now I put the UDF class in its own file...

And the test_funnel_persons.ambr still has all the TestFunnelPersonsUDF stuff and test_funnel_persons_udf.ambr has TestFunnelPersons in it.

:exploding_head: